geometry_msgs/Pose goal_pose
---
bool success